src/java/org/apache/cassandra/service/ActiveRepairService.java

```diff
@@ -304,6 +306,12 @@ public int getRepairSessionSpaceInMegabytes()
         return stats;
     }
 
+    @Override
+    public int parentRepairSessionCount()
```

Comment:
nit which I understand is not part of this patch and you override the method but I would personally go ahead and change it to `parentRepairSessionsCount()`